week-1/7-css-box/styles.css

```diff
@@ -1,6 +1,13 @@
 /* Try different box model properties below */
 .pages__page {
+  display: inline-block;
+  padding: default;
```

Comment:
I don't think `padding: default` is valid CSS. Did you find this somewhere?

Reply:
i don't know i think it was already written or may be written by mistake